src/main/webapp/app/shared/user-settings/ssh-settings/ssh-user-settings.component.scss

```diff
@@ -0,0 +1,125 @@
+textarea {
+    width: 600px;
+    height: 150px;
+    font-size: small;
+    font-family:
+        Bitstream Vera Sans Mono,
+        Courier New,
+        monospace;
+}
+
+table {
+    width: 100%;
+    border-collapse: collapse;
+    margin: 20px 0;
+    font-size: 16px;
+    text-align: left;
+}
+
+th,
+td {
+    padding: 15px 15px;
+}
+
+/* Button styling */
+
+.btn {
+    border-radius: 0;
+}
+
+.container {
+    position: relative;
+}
+
+.icon-column {
+    width: auto;
+    margin-right: 15px;
+}
+
+.vertical-center {
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translateY(-50%);
+    transform: translateY(-50%);
+}
+
+tbody tr:hover {
+    background-color: var(--ssh-key-table-hover-background);
+}
+
+/* dd container */
+.dropdown {
+    display: inline-block;
+    position: relative;
+    outline: none;
+    margin: 0;
+}
+
+/* button */
+.dropbtn {
+    padding: 0 0;
+    color: grey;
+    cursor: pointer;
+    transition: 0.35s ease-out;
+}
+
+/* dd content */
+.dropdown .dropdown-content {
+    position: absolute;
+    top: 50%;
+    min-width: 120%;
+    box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+    z-index: 100000;
```

Comment:
Maybe add a comment here explaining why we need to set the z-index to a high value